### DISCUSSION TITLE
umi4 单页切换的Suspense loading 配置缺失

### DISCUSSION THREAD

**Discussion by cailongl** [umijs/umi#11028] [Q&A]
![image](https://user-images.githubusercontent.com/21746104/233985992-a476b727-cdb3-44a0-8538-7983c76fffa3.png)
这里源码里已经有对应的配置了，但是没有暴露给用户配置，导致现在单页切换加载时，页面一直是空白，体验不好，希望能把这个配置暴露出来。

**Comment by fz6m** [comment]
全局 loading 就是 `src/loading.tsx` 这个文件，默认都会用这个。

**Reply by cailongl** [reply]
好嘞，有了，thanks！
